src/environmentPath.ts

```diff
@@ -64,8 +65,33 @@ export class Environment {
   public FOLDER_SNIPPETS: string = null;
 
   constructor(private context: vscode.ExtensionContext) {
+    this.context.globalState.update("_", undefined); // Make sure the global state folder exists. This is needed for using this.context.globalStoragePath to access user folder
+
+    this.isPortable = !!process.env.VSCODE_PORTABLE;
+
+    this.OsType = process.platform as OsType;
+    if (!this.isPortable) {
+      this.PATH = resolve(this.context.globalStoragePath, "../../..").concat(

```

Comment:
As far as I know, `path.resolve` does this automatically. However, I'll make sure to do additional testing to make sure. I have tested it on Linux, so it will definitely work on macOS.

Reply:
After testing on Windows, I can conclude that `path.resolve` in conjunction with `path.normalize` completely solves this potential issue.